### PR TITLE
Update set-up-anti-phishing-policies.md

### PR DESCRIPTION
**Pull request by urgandhi**
updated the documentation for more emphasis of the first contact safety tip as it is a strong recommendation and adoption is increasing drastically (~10K tenants so far)

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md [hunk 1]
```diff
@@ -204,7 +204,8 @@ The following impersonation settings are only available in anti-phishing policie
 
   > [!IMPORTANT]
   >
-  > Even when the impersonation safety tips are turned off, **we recommended** that you use a mail flow rule (also known as a transport rule) to add a message header named **X-MS-Exchange-EnableFirstContactSafetyTip** with value **enable** to messages. A safety tip will notify recipients the first time they get a message from the sender or if they don't often get messages from the sender.
+  > **Recommendation for enabling safety tip that will appear based on first time contact between sender and recipient(s)**:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md]:
```suggestion
  > **Recommendation for enabling a safety tip that will appear during the first-time contact between the sender and recipient(s)**:
```

File: microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md [hunk 2]
```diff
@@ -204,7 +204,8 @@ The following impersonation settings are only available in anti-phishing policie
 
   > [!IMPORTANT]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md:206]:
```suggestion

    > [!IMPORTANT]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**